tools/ci/log4j.properties

```diff
@@ -73,3 +73,8 @@ logger.consumer.level = OFF
 logger.queryablestate.name = org.apache.flink.queryablestate
 logger.queryablestate.level = TRACE
 logger.queryablestate.appenderRef.out.ref = ConsoleAppender
+
+# Temporarily enable TRACE for SourceReaderBase to debug a deadlock
+logger.sourcereader.name = org.apache.flink.connector.base.source.reader
+logger.sourcereader.level = TRACE
```

Comment:
Are we going to keep this trace logging?

Reply:
We could (it does not affect any other tests) but I was planning to remove this once we are confident with the implementation.

Reply:
Let me remove it during merging. We can always add it back if needed.